build/docker/jmxtrans-agent.xml

```diff
@@ -0,0 +1,20 @@
+<jmxtrans-agent>`
+    <queries>
+        <query objectName="metrics:name=counterPitchforkSpans.protocol.http.transport.jsonv1" attributes="FifteenMinuteRate,MeanRate,OneMinuteRate,RateUnit" resultAlias="counterPitchforkSpans.protocol.http.transport.jsonv1.#attribute#"/>
+        <query objectName="metrics:name=counterPitchforkSpans.protocol.http.transport.jsonv2" attributes="FifteenMinuteRate,MeanRate,OneMinuteRate,RateUnit" resultAlias="counterPitchforkSpans.protocol.http.transport.jsonv2.#attribute#"/>
+        <query objectName="metrics:name=counterPitchforkSpans.protocol.http.transport.protobuf" attributes="FifteenMinuteRate,MeanRate,OneMinuteRate,RateUnit" resultAlias="counterPitchforkSpans.protocol.http.transport.protobuf.#attribute#"/>
+        <query objectName="metrics:name=counterPitchforkSpans.protocol.http.transport.thrift" attributes="FifteenMinuteRate,MeanRate,OneMinuteRate,RateUnit" resultAlias="counterPitchforkSpans.protocol.http.transport.thrift.#attribute#"/>
+        <query objectName="metrics:name=counterPitchforkSpansInvalid.invalid.true" attributes="FifteenMinuteRate,MeanRate,OneMinuteRate,RateUnit" resultAlias="counterPitchforkSpansInvalid.invalid.true.#attribute#"/>
+        <query objectName="metrics:name=httpServerRequests.exception.None.method.POST.outcome.SUCCESS.status" attributes="FifteenMinuteRate,MeanRate,OneMinuteRate,RateUnit" resultAlias="httpServerRequests.exception.None.method.POST.outcome.SUCCESS.status.#attribute#"/>
+        <query objectName="metrics:name=logbackEvents.level.warn" attributes="FifteenMinuteRate,MeanRate,OneMinuteRate,RateUnit" resultAlias="logbackEvents.level.warn.#attribute#"/>
+    </queries>
+
+    <outputWriter class="org.jmxtrans.agent.GraphitePlainTextTcpOutputWriter">
+        <host>${GRAPHITE_HOST:localhsot}</host>
```

Comment:
There's a typo here: `localhsot` -> `localhost`.
Is it possible to avoid default values for these properties though? Cause if property resolution fails there's no feedback.